### PR TITLE
fix(web-app): show library name/version inside nav, sidenav refactor

### LINKED ISSUE #523: add real data to library side nav
Rel #464 
Rel #509 

Library sidenav needs to be wired up to data
Screen Shot 2022-04-08 at 7 03 08 AM

### LINKED ISSUE #560: Link to library assets page from the library dashboard
The library detail page dashboard currently says "Coming soon..."

- [x] Update button text to "View assets"
- [x] Update button href to `/library-assets` within a library (the library assets page, whatever that route ends up being)

Reference, in B:

image

### PR DESCRIPTION
**Pull request by francinelucca**
Closes #523 
Closes #560 

#### New

- Function to obtain library side nav data

#### Changed

- nav-main -> nav-primary
- nav-library -> nav-secondary
- layout refactor
- moved a lot of custom nav (primary,secondary) logic/styling to layout 

### Removed

- removed unstyled list of assets in library detail 
image
- Mocked content from library side nav

#### Testing / reviewing

Test that main nav and library nav work as expected and that library side nav maps correctly to library name and version and list of assets (design kits and versions show "Page Not Found" for now since we don't have those pages yet).

### CODE REVIEW

**Comment by alisonjoseph** [comment]
I don't think the left nav should show up on this page?

Screen Shot 2022-05-31 at 2 54 35 PM

**Comment by alisonjoseph** [comment]
View assets button sends me to a page not found

http://localhost:3000/assets/ibmdotcom-web-component
Screen Shot 2022-05-31 at 2 57 12 PM
s

File: services/web-app/layouts/layout/layout.module.scss [hunk 1]
```diff
@@ -62,15 +63,74 @@
...
+}
+
+.side-nav-slide {
+  @include breakpoint.breakpoint(lg) {
+    display: flex;
+    width: 32rem;
+    height: 100%;
+    transform: translateX(0);
+
+    /* I don't know why it's complaining if I remove these two disable lines */
```
Comment (alisonjoseph) [inline services/web-app/layouts/layout/layout.module.scss:103]: weird. I'll take a look

**Comment by francinelucca** [comment]
> I don't think the left nav should show up on this page?
> 

Ahhh my bad, fixed!

**Comment by francinelucca** [comment]
> View assets button sends me to a page not found
> 

Nice! fixed!!

**Review by alisonjoseph — DISMISSED** [review]
LGTM! Everything seems to be working as intended 🥳 

Just the small adjustment to the scss file for reduced motion linting

File: services/web-app/layouts/layout/layout.module.scss [hunk 2]
```diff
@@ -62,15 +63,74 @@
...
+    &.slide-out {
+        transition: none;
+      }
+    }
+
+    &.slide-in,
+    /* stylelint-disable-next-line a11y/media-prefers-reduced-motion */
+    &.slide-out {
+      transition: transform motion.$duration-moderate-01 motion.motion(entrance, productive);
+    }
```
[the remark this answers is not in the document]
Comment (francinelucca) [reply services/web-app/layouts/layout/layout.module.scss]: thanks!!

**Review by alisonjoseph — APPROVED** [review]
👍

**Review by andreancardona — APPROVED** [review]
🔥 🔥 double fire!

**Pull request merged**